sources/payment-hub/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Sun Aug 25 22:55:28 IST 2019
+version=1.0.0-SNAPSHOT
+groupId=hu.dpc.paymenthub
```

Comment:
Again, looks like a build script generated file which shouldn't be added to source control.